src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderSentenceResource.java

```diff
@@ -99,6 +101,7 @@ public List<OffenderSentenceCalc<BaseSentenceCalcDates>> getOffenderSentencesHom
     })
     @Operation(summary = "Retrieve the current state of the latest Home Detention Curfew for a booking")
     @GetMapping("/booking/{bookingId}/home-detention-curfews/latest")
+    @VerifyBookingAccess(overrideRoles = "VIEW_PRISONER_DATA")
```

Comment:
only called by auth code - but have left alternative override role in to match the other